sdk/spring/azure-spring-data-cosmos/src/test/java/com/azure/spring/data/cosmos/core/CosmosTemplateIT.java

```diff
@@ -212,6 +218,18 @@ public void testFindAll() {
         assertThat(responseDiagnosticsTestUtils.getCosmosResponseStatistics().getRequestCharge()).isGreaterThan(0);
     }
 
+    @Test
+    public void testFindByIdPointRead() {
```

Comment:
This doesn't seem to be testing anything unique to show that it is in fact doing the point read and not cross-partition, is there anything to test to be sure it isn't doing a cross-partition query? Maybe here we can test that the warning message added isn't logged and in testFindById() below we can test that the warning message is logged? The same goes for the reactive tests.

Reply:
Well, this test is using a new entity where id is explicitly defined as partitionKey. Since that’s the only scenario in which findById will execute a point read, it will have to be a point read. But yeah, adding a test for the warning message might be better.